lib/docs/src/components/table.stories.tsx

```diff
@@ -267,6 +267,18 @@ export const SelectableMultiple = (): JSX.Element => {
 	);
 };
 
+Utils.story(SelectableMultiple, {
+	desc: `
+To choose multiple rows from the table, user can use the \`selectable\` prop.
```

Comment:
nah, "user" is not the one the use the "selectable" prop. We have 2 actors here: "users" mean the end users that will use the app. "Developers" mean the users of our ui kit, the ones who build the app. We usually don't say "developers" explicitly, but mention them as 2nd party ("you").

This sentence is not really correct because "users" are the ones to "choose rows", but "developers" are the one to set the prop. I suggest to change it to something like "To let users select some rows of a table, provide the "selectable" prop"